sources/packages/api/src/services/application/application.service.ts

```diff
@@ -119,6 +128,15 @@ export class ApplicationService extends RecordDataModelService<Application> {
         APPLICATION_NOT_VALID,
       );
     }
+
+    const now = new Date();
+    const auditUser = { id: auditUserId } as User;
+    const originalAssessment = new StudentAssessment();
+    originalAssessment.triggerType = AssessmentTriggerType.OriginalAssessment;
+    originalAssessment.submittedBy = auditUser;
+    originalAssessment.submittedDate = now;
```

Comment:
since the date is a timestamp, its better to fed directly right? same in line 153, my thought

Reply:
Why being a timestamp should be a justification to use the new Date()? Do you mean if between the time of one new Date() and another there will be some milliseconds of difference, right?
From DB's perspective, it seems better to me if the date is expressed as the same. We are capturing the time that the record was created and not the time that it was generated in the server, does it make sense?

Reply:
I guess, the use of constant here is to match the milliseconds between the assessment submitted date and applicationStatusUpdatedDate.

Reply:
Talked with @ann-aot and I believe that we are good about it.
Yes @guru-aot, that is the idea 😉